activerecord/test/cases/adapters/postgresql/datatype_test.rb

```diff
@@ -44,6 +44,13 @@ def test_time_values
     assert_equal (-21.day), @first_time.scaled_time_interval
   end
 
+  def test_update_large_time_in_seconds
```

Comment:
I was just curious and anyone else wondering this test fails prior to this patch with:

```
Error:
PostgresqlDataTypeTest#test_update_large_time_in_seconds:
ActiveRecord::StatementInvalid: PG::IntervalFieldOverflow: ERROR:  interval field value out of range: "PT2208986640.0S"
CONTEXT:  unnamed portal parameter $1 = '...'
```